spec/system/task_spec.rb

```diff
@@ -1,24 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project){ create(:project) }
+  let(:task){ create(:task, project: project) }
```

Comment:
ProjectのFactoryで`association`を設定することで、この記載は省略できますね。
```suggestion
  let(:task){ create(:task) }
```

Reply:
[3fa6615](https://github.com/yoshiki105/rspec_app_exam/pull/1/commits/3fa66153c4981a616f3d95e9b5ff922c9475e2dc) でtasks.rbにassociationを追加しました。